pytorch_lightning/trainer/properties.py

```diff
@@ -59,23 +60,92 @@ class TrainerProperties(ABC):
     _default_root_dir: str
     _weights_save_path: str
     accelerator_backend: Accelerator
-    logger: LightningLoggerBase
-    model_connector: ModelConnector
-    checkpoint_connector: CheckpointConnector
-    callbacks: List[Callback]
     num_nodes: int
     num_processes: int
+    accelerator_connector: BackendConnector
     _lightning_optimizers = None
 
+    @property
+    def accelerator(self):
+        return self.accelerator_connector.accelerator
+
```

Comment:
type hints here would be helpful

Reply:
Will be added. Aiming for complete mypy coverage in the near future:)